rxjava-modules/rxjava-observables/src/test/java/com/baeldung/rxjava/combine/ObservableMergeVsConcatUnitTest.java

```diff
@@ -0,0 +1,129 @@
+package com.baeldung.rxjava.combine;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+
+import rx.Observable;
+import rx.observers.TestSubscriber;
+
+public class ObservableMergeVsConcatUnitTest {
+
+    @Test
+    public void givenTwoSynchronousObservables_whenConcatenated_thenAllValuesEmittedInOrder() {
+
```

Comment:
Omit the newline at the start of all of the test methods
```suggestion
```

Reply:
Fixed